app/models/bus_stop.rb

```diff
@@ -207,7 +208,13 @@ def route_list
   def self.to_csv(limited_attributes: false)
     attrs = if limited_attributes
               LIMITED_ATTRS
-            else Hash[columns.map { |c| [c.name, c.name.humanize] }]
+            else
+              hashed_columns = Hash[columns.map { |c| [c.name, c.name.humanize] }]
+              duplicates = %w[name hastus_id updated_at created_at route_list]
+              duplicates.each do |v|
+                hashed_columns = hashed_columns.except(v)
+              end
+              LIMITED_ATTRS.merge hashed_columns
```

Comment:
Getting there - forgot that we need the humanized names, so just `column_names` won't do.

Instead of the `duplicates = ` and then `duplicates.each` I would suggest

```ruby
hashed_columns = Hash[...].except('name', ...)
```

Reply:
You can see I've been getting carried away with my `each` loops lately.